mantidproject/mantid
# LoadILLSANS

This upgrades the SANS loader to support the modern format for D11, D22 and D33 instruments at ILL.

**To test:**



Load the 4 nexus files you find in unit test data under ILL/D11, ILL/D22, ILL/D33 (set facility to ILL first).
Check that the output has the instrument attached (right click->show instrument).
The rest is captured by the unit tests.

Fixes #22212 . 

Also merge PR [#109](https://github.com/mantidproject/mantidgeometry/pull/109) in geometry.

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
